images/base/files/etc/containerd/config.toml

```diff
@@ -19,11 +19,18 @@ version = 2
   runtime_type = "io.containerd.runc.v2"
   # Generated by "ctr oci spec" and modified at base container to mount poduct_uuid
   base_runtime_spec = "/etc/containerd/cri-base.json"
+  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
+    # use systemd cgroup by default
+    SystemdCgroup = true
 
 # Setup a runtime with the magic name ("test-handler") used for Kubernetes
 # runtime class tests ...
 [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.test-handler]
+  # same settings as runc
```

Comment:
What is this test-handler used for?

Reply:
see comment two lines above in file? :-)

Reply:
https://github.com/kubernetes/kubernetes/blob/e045e5c875b2f51c4c4b1bc14d4e717123649b17/test/e2e/framework/node/runtimeclass.go

it's a tricky problem in k/k e2e testing. to test that runtimeclass works, but need to know that a non-default runtime is configured.

Reply:
I was wondering why do we need to fill this too, but seems that is for coherence,

Reply:
found it in the 5th commit

Reply:
in e2e we will actually create pod(s) with this runtime, we need that to match kubelet wrt cgroup driver